code/__HELPERS/unsorted.dm

```diff
@@ -1053,8 +1048,7 @@ proc/GaussRandRound(var/sigma,var/roundto)
 					X.icon_state = old_icon_state1
 					X.icon = old_icon1 //Shuttle floors are in shuttle.dmi while the defaults are floors.dmi
 
-					if(new_air)
-						X.air = new_air
+					X.return_air().copy_from(T.return_air())
```

Comment:
I don't think this always works. Just use `X.air` when assigning to it before there's a zone (such as here)

Reply:
Actually it does always work but it's still weird

Reply:
so do I change it or not?

Reply:
I guess this way is a bit more robust, if rather weird